### PR TITLE
Allow to fill extent outside Swiss projection extent

### PR DESCRIPTION
**Pull request by tonio**
Fixes Jira : GEOCAT_SB-431

### CODE REVIEW

File: web-ui/src/main/resources/catalog/geocat-shared-objects/js/extent-directive.js [hunk 1]
```diff
@@ -122,23 +128,38 @@
...
                     scope.formObj.proj, map.getView().getProjection());
-                coordinates = gnMap.getPolygonFromExtent(extentProj);
-                geom = new ol.geom.Polygon(coordinates);
 
-                f = new ol.Feature();
-                f.setGeometry(geom);
-                f.getGeometry().setCoordinates(coordinates);
-                featureOverlay.addFeature(f);
+                scope.incompatible = mapProj != scope.formObj.proj &&
+                  !ol.extent.containsExtent(mapProj.getExtent(), scope.extent);
```
Comment (fgravin) [inline web-ui/src/main/resources/catalog/geocat-shared-objects/js/extent-directive.js]: `mapProj.getExtent()` and `scope.extent` in the same proj ??

File: web-ui/src/main/resources/catalog/geocat-shared-objects/js/extent-directive.js [hunk 2]
```diff
@@ -89,8 +89,14 @@
...
                */
               scope.fillInput = function() {
-                var geom = featureOverlay.getFeatures().item(0).getGeometry().
-                    clone().transform(map.getView().getProjection(), scope.formObj.proj);
+                var feature = featureOverlay.getFeatures().item(0) || scope.feature;
+                if (!feature) {
+                  return;
+                }
+                var geom = feature.getGeometry().clone();
+                if (!scope.incompatible) {
```
Comment (fgravin) [inline web-ui/src/main/resources/catalog/geocat-shared-objects/js/extent-directive.js]: Why in `scope` ?

Comment (fgravin) [inline web-ui/src/main/resources/catalog/geocat-shared-objects/js/extent-directive.js]: Do you set it back to `false` somehow, for example if afterwards the user draws a bbox ?

**Comment by tonio** [comment]
@fgravin PR updated:
 - Set `incompatible` to false on `boxend`
 - Use reprojected extent

**Comment by fgravin** [comment]
Looks good thanks

**Pull request merged**